src/index.html

```diff
@@ -1,10 +1,12 @@
 <html>
   <head>
     <link rel="stylesheet" type="text/css" href="style.css" />
-    <script src="script.js"></script>
+    <script type="text/javascript" src="combinations.js"></script>
+    <script type="text/javascript" src="script.js"></script>
   </head>
   <body>
     <h1>Sensei Kimura's Fighting Combinations</h1>
+    <h6>Welcome Let's Get Started</h6>
```

Comment:
heading levels should be hierarchical.  if you don't like the style, don't use h6, restyle h2